docs/Catalog/RemovingStacks.md

```diff
@@ -0,0 +1,100 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+If you ever need to delete a Quilt stack from your AWS account, you should know
+that:
+
+- Your data in S3 buckets remains accessible.
+- You can continue using the Quilt SDK for reading and writing packages.
+- Stack-managed resources (Athena tables, user database, audit logs) will be affected.
+- The deletion behavior varies between CloudFormation and Terraform deployments.
+
+> **Consider Pausing Instead:** To temporarily suspend the stack, you can shut
+> down ECS services to reduce costs. Note that some charges may continue even
+> when ECS is inactive, so the only way to completely eliminate all costs is to
+> delete the stack.
+
+---
+
+## I. Impact Assessment and Preparation
+
+### What May Be Deleted
+
+- Custom configurations specific to the Quilt stack (e.g., IAM roles, resource
+  mappings, permissions).
+- Analytics data in stack-managed buckets.
+- Audit logs and Athena querying setups.
+- User database configurations.
+
+### Recommended Actions
+
+- Export analytics data you need to keep.
+- Save important audit logs.
+- Document existing Athena configurations if you'll need to recreate them.
+- If using Terraform, use it to capture a snapshot of the user database for
```

Comment:
I guess even if you don't use terraform you still might want to make snapshot of DB to reuse it *somehow* (e.g. we could add option to restore from snapshot for CFN deployment or they might want to switch to terraform deployment)

Reply:
> user database

I'm not sure what you exactly mean

Reply:
Is this better?

Reply:
not quite
I mean users is not the only thing stored by Postgres (e.g. tabulator tables config)

Reply:
Updated